Silicon/AlderlakePkg/Include/Library/HeciMeExtLib.h

```diff
@@ -98,4 +99,32 @@ HeciGetHerHashData (
   IN UINT8     HerMaxRegCount
   );
 
+/**
+  Send Set FIPS Mode to Enabled or Disabled
+
+  @retval EFI_UNSUPPORTED         Current ME mode doesn't support this function
+  @retval EFI_SUCCESS             Command succeeded
+  @retval EFI_DEVICE_ERROR        HECI Device error, command aborts abnormally
+  @retval EFI_TIMEOUT             HECI does not return the buffer before timeout
+  @retval EFI_BUFFER_TOO_SMALL    Message Buffer is too small for the Acknowledge
+**/
+EFI_STATUS
+HeciSetFipsMode (
```

Comment:
Add "EFIAPI" modifier for the library APIs in both header file and implementation.